crates/weaver_forge/src/lib.rs

```diff
@@ -497,6 +497,9 @@ mod tests {
                     expected_dir.as_ref().join(file),
                     observed_dir.as_ref().join(file)
                 );
+
+                eprintln!("Expected file content: {}", file1_content);
```

Comment:
FYI - you can use the diff library I pulled for this purpose:

https://github.com/open-telemetry/weaver/blob/main/crates/weaver_semconv_gen/src/diff.rs#L12

We can move that somewhere more friendly if needed.

Reply:
Good point. Created an issue for a future PR. 
https://github.com/open-telemetry/weaver/issues/77